src/context/WalletProvider/SelectModal.tsx

```diff
@@ -1,16 +1,17 @@
 import { Button, Image, ModalBody, ModalHeader, Stack, Text } from '@chakra-ui/react'
-import { RawText } from 'components/Text'
+import { RawText, Text as Text1 } from 'components/Text'
 
 import { SUPPORTED_WALLETS } from './config'
 
 export const SelectModal = ({ connect }: { connect: (adapter: string) => Promise<void> }) => {
   return (
     <>
-      <ModalHeader>Connect a wallet</ModalHeader>
+      <ModalHeader>
+        <Text1 translation={'walletProvider.sModal.header'} />
+      </ModalHeader>
       <ModalBody>
         <RawText mb={6} color='gray.500'>
-          By connecting a wallet, you agree to ShapeShifts’ Terms of Service and acknowledge that
-          you have read and understand the ShapeShift disclaimer.
+          <Text1 translation={'walletProvider.sModal.body'} />
```

Comment:
Same here can we get rid of the chakra Text with RawText and update Text1 to be Text?